### ISSUE TITLE
Resource names with a space in them are line split in markdown, breaking formatting

### ISSUE THREAD

**Issue by robzr** [dineshba/tf-summarize#82]
Example of generated markdown from 0.3.13. Note that this is not a regression, this has been happening as long as I've used `tf-summarize`:
```
| CHANGE |                      RESOURCE                       |
|--------|-----------------------------------------------------|
| add    | `github_organization_ruleset.this["Org              |
|        | Require Signatures"]`                               |
|        | `github_repository_ruleset.this["repo-next/Code     |
|        | Freeze"]`                                           |
```
Renders as:
| CHANGE |                      RESOURCE                       |
|--------|-----------------------------------------------------|
| add    | `github_organization_ruleset.this["Org              |
|        | Require Signatures"]`                               |
|        | `github_repository_ruleset.this["repo-next/Code     |
|        | Freeze"]`                                           |

**Comment by dineshba** [comment]
Thanks for reporting this issue @robzr . I was able to reproduce this issue.

If the resource name has space in-between and if it crossed certain length, then this auto-wrapping happen and goes into new line which creates invalid table row syntax.

Doesn't look like quick fix. I will try to fix it soon. [Code location](https://github.com/dineshba/tf-summarize/blob/main/writer/table.go#L21-L35).  If anyone able to fix it, please feel free to raise PR

**Comment by wobondar** [comment]
@dineshba please have a look on PR #83

**Comment by dineshba** [comment]
Released as part of v0.3.14 version... @robzr thanks for reporting and @wobondar for fixing it
